src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -525,100 +536,21 @@
 
             bool succeeded = true;
 
-            succeeded &= ValidateFile(context, options.OutputFilePath, shouldExist: null);
-            succeeded &= ValidateFile(context, options.ConfigurationFilePath, shouldExist: true);
-            succeeded &= ValidateFiles(context, options.PluginFilePaths, shouldExist: true);
-            succeeded &= ValidateFile(context, options.BaselineSarifFile, shouldExist: true);
+            succeeded &= ValidateFile(context, options.OutputFilePath, DefaultPolicyName, shouldExist: null);
+            succeeded &= ValidateFile(context, options.ConfigurationFilePath, DefaultPolicyName, shouldExist: true);
+            succeeded &= ValidateFiles(context, options.PluginFilePaths, DefaultPolicyName, shouldExist: true);
+            succeeded &= ValidateFile(context, options.BaselineSarifFile, DefaultPolicyName, shouldExist: true);
             succeeded &= ValidateInvocationPropertiesToLog(context, options.InvocationPropertiesToLog);
             succeeded &= ValidateOutputFileCanBeCreated(context, options.OutputFilePath, options.Force);
             succeeded &= options.ValidateOutputOptions(context);
+            succeeded &= ValidateExportUri(options.OutputFilePath, options.ExportUri);
```

Comment:
>ValidateExportUri [](http://example.com/codeflow?start=25&length=17)

Don't add this helper, instead update ValidateOutputOptions. Posting to a URI is a new kind of output.

Reply:
removed the method and updated this. thanks for the suggestion :)